skxray/img_proc/mathops.py

```diff
@@ -0,0 +1,349 @@
+# Module for the BNL image processing project
+# Developed at the NSLS-II, Brookhaven National Laboratory
+# Developed by Gabriel Iltis, Oct. 2013
+"""
+This module is designed to facilitate image arithmetic and logical operations
+on image data sets.
+"""
+
+import numpy as np
+import parser
+
+
+def arithmetic_basic(input_1,

```

Comment:
The functions logic_basic, arithmetic_basic and arithmetic_custom have been renamed as logic, arithmetic, and arithmetic_expression as we discussed. Bullets 2,3,and 4 were addressed in previous commits and comments.

All is good in the world. Well, my world at least....